cmd/templates-api/main.go

```diff
@@ -34,12 +34,20 @@ func main() {
 	// Flags.
 	httpHostF := flag.String("http-host", "0.0.0.0", "HTTP host")
 	httpPortF := flag.String("http-port", "8000", "HTTP port")
+	repositoryPathF := flag.String("repository-path", "https://github.com/keboola/keboola-as-code-templates.git:api-demo", "Alternative path to default repository")
 	debugF := flag.Bool("debug", false, "Log request and response bodies")
 	flag.Parse()
 
 	// Setup logger.
 	logger := log.New(os.Stderr, "[templatesApi]", 0)
 
+	if *repositoryPathF != "" {
```

Comment:
`if *repositoryPathF != "" {` toto by tu nemalo byt, ked to ma default hodnotu? ... ako mozes manualne zadat `--repository-path ""`, len to by malo skoncit chybou.

Reply:
omg, no jo 🙈